package.json

```diff
@@ -46,7 +46,7 @@
     "@vue/cli-plugin-router": "~4.5.0",
     "@vue/cli-plugin-typescript": "~4.5.0",
     "@vue/cli-plugin-unit-jest": "~4.5.0",
-    "@vue/cli-service": "~4.5.0",
+    "@vue/cli-service": "4.5.19",
```

Comment:
Can you add some information about this change?

Reply:
Sure

Reply:
It was done when I downloaded the node.js 14.10 version.